### PR TITLE
Async support

### PR DESCRIPTION
**Pull request by fredrike**

### CODE REVIEW

**Comment by mellado** [comment]
Why did you add a "_" to the _send in the async version and no "_" for async update function?

**Comment by fredrike** [comment]
> Why did you add a `_` to the _send in the async version and no `_` for async update function?

I made the send method "private" as it is internal and should just be used by other class members. https://docs.python.org/3/tutorial/classes.html#private-variables

File: tfiac.py [hunk 1]
```diff
@@ -62,7 +62,6 @@ def __init__(self, host):
         self._status = {}
         self._name = None
         self._available = True
-        self.update()
```
Comment (mellado) [inline tfiac.py]: Is it not needed to call 'update' at the beginning anymore?

Comment (mellado) [reply tfiac.py]: Now you get the following error:

> Traceback (most recent call last):
>   File "/Users/pavlom/Projects/home-assistant/homeassistant/helpers/entity_platform.py", line 352, in _async_add_entity
>     await entity.async_update_ha_state()
>   File "/Users/pavlom/Projects/home-assistant/homeassistant/helpers/entity.py", line 232, in async_update_ha_state
>     state = self.state
>   File "/Users/pavlom/Projects/home-assistant/homeassistant/components/climate/__init__.py", line 205, in state
>     if self.is_on is False:
>   File "/Users/pavlom/Projects/home-assistant/homeassistant/components/climate/tfiac/__init__.py", line 116, in is_on
>     self._client.status[ON_MODE])
> KeyError: 'is_on'

Comment (fredrike) [reply tfiac.py]: Yes and we are: https://github.com/fredrike/home-assistant/blob/ab864c0d84427ce38f95b73bf97671e4c2b9ac90/homeassistant/components/climate/tfiac.py#L61

We are however (afaik) not allowed to do async tasks in init.

**Pull request merged**